.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
settings file not a issue, this can be useful if people are using special extensions in VSCODE and enhances extension usage